src/views/ApplicationViews.js

```diff
@@ -6,6 +6,7 @@ import { Posts } from "../components/posts/Posts"
 import { MyPosts } from "../components/posts/MyPosts"
 import { Categories } from "../components/categories/Categories"
 import { Tags } from "../components/tags/Tag"
```

Comment:
I'm receiving an error because the import. Could you update the name for line 8 & line 27?

Reply:
I must have accidentally pushed a former version. I'll go fix that.